python/ray/serve/_private/http_proxy.py

```diff
@@ -110,6 +119,12 @@
 INITIAL_BACKOFF_PERIOD_SEC = 0.05
 MAX_BACKOFF_PERIOD_SEC = 5
 BACKOFF_FACTOR = 2
+drained_message = "This node is being drained."
+success_message = "success"
+
+
+def generate_request_id() -> str:
+    return str(uuid.uuid4())
 
 
 class GenericProxy(ABC):
```

Comment:
do we also need to update `serve_num_ongoing_http_requests`?

Reply:
Oops thanks for catching it, but got in when I was rebasing!